### PR TITLE
Readme improvements w/ Cyber Security sheet

### PR DESCRIPTION
**Pull request by martysweet**
Improves the documentation around how this application works.

Adds a sheet for Cyber Security

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,43 @@ Reload all permissions:
...
+### Bertha
+
+Bertha is an application which converts a Google Spreedsheet into JSON or other formats. It is used in this
+application to read the spreadsheets which are then used to apply the permissions to Heroku. 
+Google sheets used with this application should be shared with the "Anyone at FT with the link can view".
+For more information see [Bertha](https://github.com/ft-interactive/bertha/wiki/Tutorial) on Github.
+
+### App Permissions
+
+The following list provides a mapping between a team and their apps. Users within the `devs` tab will gain permissions on apps stated within the `apps` tab.
```
Comment (JakeChampion) [inline README.md]: Next has a different set up, here is their sheet for devs - `https://docs.google.com/spreadsheets/d/1mbJQYJOgXAH2KfgKUM1Vgxq8FUIrahumb39wzsgStu0/edit#gid=0`

Comment (JakeChampion) [inline README.md]: Here is the sheet for Next extra apps - `https://docs.google.com/spreadsheets/d/1JcsgN_YLG67KXqRQi-vRloUgvywj7BWdg4rt4OoGsDw/edit#gid=0`

Comment (JakeChampion) [inline README.md]: And here is where Next get the rest of their apps from - `https://next-registry.ft.com/v2/`

Comment (martysweet) [inline README.md]: Fixed on line 34

**Pull request merged**